content/docs/testing-farm.md

```diff
@@ -7,8 +7,7 @@ aliases: [/testing-farm]
 
 # Testing Farm
 
-Testing Farm is Packit's testing system.
-It is a containerized service for running tests.
+[Testing Farm][tf] is Packit's testing system.
 Test execution is configured via [Flexible Metadata Format][fmf],
 according to the [Metadata Specification][spec].
```

Comment:
Maybe we could just mention `tmt` here instead of of `fmf` and metadata spec which might be confusing.